recipe/laravel.php

```diff
@@ -35,6 +35,16 @@
     'storage/logs',
 ]);
 
+set('laravel_version', function () {
+    $result = run('{{bin/php}} {{release_path}}/artisan --version');
+
+    preg_match_all('/version\ (.+?)$/', $result, $matches);
+
+    $version = $matches[1][0] ?? 5.4;
+
+    return (float) $version;
```

Comment:
@barryvdh maybe I can change the regex to return the _right_ version, but I did this because `5.2.13` > `5.2`, so we can get a false positive. What do you think?